R/get_regional_data.R

```diff
@@ -116,38 +118,37 @@ get_regional_data <- function(country, totals = FALSE, include_level_2_regions =
   # Totalise and return if totals data is requested ----------------------------------
   if (totals) {
     data <- totalise_data(data, include_level_2_regions = include_level_2_regions) %>%
-      dplyr::arrange(-cases_total) %>%
-      rename_region_column(country) %>%
-      rename_region_code_column(country)
-    return(tibble::tibble(data))
-  }
-
-  # Pad the data set ------------------------------------------------------------------
-  data <- data %>%
-    tidyr::drop_na(date) %>%
-    fill_empty_dates_with_na() %>%
-    complete_cumulative_columns()
-
-  # Select and arrange the data -------------------------------------------------------
-  if (include_level_2_regions) {
-    data <- data %>%
-      dplyr::select(date, region_level_2, level_2_region_code, region_level_1, level_1_region_code, 
-                    cases_new, cases_total, deaths_new, deaths_total,
-                    recovered_new, recovered_total, hosp_new, hosp_total,
-                    tested_new, tested_total) %>%
-      dplyr::arrange(date, region_level_1, region_level_2)
-  } else {
+      dplyr::arrange(-cases_total)
+  }else{
+    # Pad the data set ------------------------------------------------------------------
     data <- data %>%
-    dplyr::select(date, region_level_1, level_1_region_code, cases_new, cases_total, deaths_new, deaths_total,
-                  recovered_new, recovered_total, hosp_new, hosp_total,
-                  tested_new, tested_total) %>%
-    dplyr::arrange(date, region_level_1)
+      tidyr::drop_na(date) %>%
+      fill_empty_dates_with_na() %>%
+      complete_cumulative_columns()
+    
+    # Select and arrange the data -------------------------------------------------------
+    if (include_level_2_regions) {
+      data <- data %>%
+        dplyr::select(date, region_level_2, level_2_region_code, region_level_1, level_1_region_code, 
+                      cases_new, cases_total, deaths_new, deaths_total,
+                      recovered_new, recovered_total, hosp_new, hosp_total,
+                      tested_new, tested_total) %>%
+        dplyr::arrange(date, region_level_1, region_level_2)
```

Comment:
i think good to add `dplyr::everything()` at the end of any call to `dplyr::select` so that all other variables (in the UK data for now) get returned at the same time

Reply:
doesn't this happen in your PR (and hence not needed here)?